### PR TITLE
cleanup filecache database handling a bit

### PR DESCRIPTION
**Pull request by icewind1991**
Move building queries as string over to the query builder

### CODE REVIEW

File: lib/private/Files/Cache/Cache.php [hunk 1]
```diff
@@ -303,35 +298,39 @@ public function update($id, array $data) {
...
-		$this->connection->executeQuery($sql, $params);
+		$params = $this->buildParts($data);
 
+		$keys = array_keys($params);
+		$query = $this->connection->getQueryBuilder();
+		$query->update('filecache');
+		foreach ($keys as $key) {
+			$query->set($key, $query->createParameter($key));
+		}
+		$query->where($query->expr()->eq('filed', $query->createParameter('fileid')))
```
Comment (MorrisJobke) [inline lib/private/Files/Cache/Cache.php]: `filed` 🙈

**Review by MorrisJobke — DISMISSED** [review]
See typo

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍 Code change looks also good

**Review by rullzer — APPROVED** [review]
:elephant:

**Comment by rullzer** [comment]
Phan is not happy
https://drone.nextcloud.com/nextcloud/server/10970/274

**Comment by MorrisJobke** [comment]
Phan is still not happy :/

**Pull request closed without merging**